src/components/Notice/index.tsx

```diff
@@ -11,8 +10,9 @@ const Container = styled(Box)`
   display: flex;
   align-items: center;
 `
+
 type NoticeProps = {
-  children: ReactNode
+  children: ReactNode | string
```

Comment:
?

Reply:
my IDE who's displaying wrong warnings sorry I'm removing it

Reply:
my IDE who's displaying wrong warnings sorry I'm removing it